entryDetail.go

```diff
@@ -204,9 +204,9 @@ func (ed *EntryDetail) Parse(record string) {
 	// 2-3 is checking credit 22 debit 27 savings credit 32 debit 37
 	ed.TransactionCode = ed.parseNumField(record[1:3])
 	// 4-11 the RDFI's routing number without the last digit.
-	ed.RDFIIdentification = ed.parseStringField(record[3:11])
+	ed.RDFIIdentification = record[3:11]
```

Comment:
I think we are safe to drop the space trimming here altogether, but I can revert the changes

Reply:
Yea. It's probably fine with some fields.